x-pack/plugins/security_solution/public/detections/components/rules/query_bar/index.tsx

```diff
@@ -29,7 +29,7 @@ import * as i18n from './translations';
 export interface FieldValueQueryBar {
   filters: Filter[];
   query: Query;
-  saved_id?: string;
```

Comment:
FYI there is an option you can pass to the form **to not** strip undefined value (https://docs.elastic.dev/form-lib/core/use-form#stripemptyfields)